### PR TITLE
Flavour confidencial

### PR DESCRIPTION
**Pull request by CodingCarlos**
- Added images
 - Updated colors
 - Moved store urls to flavour configurable file
 - Fully fixed the ignore pattern for flavour components

### CODE REVIEW

File: pages/home/home.controller.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
-		.controller('HomeController', ['session', '$state', '$stateParams', Home]);
+		.controller('HomeController', ['session', '$state', '$stateParams', 'flavourString', Home]);
 
-	function Home(session, $state, $stateParams) {
+	function Home(session, $state, $stateParams, flavourString) {
 		var self = this;
 
+		self.flavour = flavourString;
+
 		var userIsLogged = session.id > 0;
```
Comment (alvaromart) [inline pages/home/home.controller.js:10]: do we need this param self.flavour?

File: pages/register/register.view.scss [hunk 2]
```diff
@@ -1,5 +1,6 @@
 .register-page {
 	@extend %align-items-center;
+	background-color: $mainColor;
```
Comment (alvaromart) [inline pages/register/register.view.scss:3]: This background-color overwrite register_bg, line below? Do we separate background login and register for vimojo (images resources) and elconfidencial (colors).

**Review by alvaromart — COMMENTED** [review]
Cool header video with el confidencial color.
We follow steps doing flavours, commit by commit +1
Review by @ruth and @alvaro

File: pages/home/home.controller.js (hunk 1 above)
Comment (CodingCarlos) [reply pages/home/home.controller.js:10]: Yes, to use it in home view, to set the right store urls.

File: pages/register/register.view.scss (hunk 2 above)
Comment (CodingCarlos) [reply pages/register/register.view.scss:3]: No, it does not override the `background-image`, but gets combined with it. If there's an image, will use the image. If not (or while the image is loading), will use the color.

**Review by alvaromart — APPROVED** [review]
Thanks for the comments, I approve code flavor

**Review by DevStarlight — APPROVED** [review]

**Pull request merged**